commons/instances.sql

```diff
@@ -1,14 +1,14 @@
-INSERT INTO user (ID, age, gender, phoneNumber, password)
-VALUES (1, 18, "female", "07734", "ovo!!!!!!");
+INSERT INTO user (ID, username, age, gender, phoneNumber, email, password)
+VALUES (1, "Dot", 18, "cis woman", "07734", "dot@example.com", "ovo!!!!!!");
```

Comment:
We should force usernames to be all lowercase on the DB side, I wrote how to do this below.

Reply:
Ah I see. I will make the corresponding changes. I realized I forgot to make my last commit and the parameters are slightly different for the create function. Instead of creating the user object in the create function, we pass in a user object as an argument in the new version. It will take a bit longer to fix because I have unfortunately added breaking changes (to implement authentication roles) as well.